tools/elk/conf/filter-marathon-1.4.x.conf

```diff
@@ -103,7 +103,21 @@ filter {
     remove_tag => ["unclassified"]
   }
   grok {
-    match => {"message" => "^Finished processing %{UUID:offerId}"}
+    match => {"message" => "^Start processing offer %{UUID:offerId}. Current offer matcher count: %{NONNEGINT:matchersCount:int}"}
```

Comment:
So we have `start`, `finish` and `no match`. Should we also add a grok for `match`? Maybe we need to adjust logging for that too?

Reply:
We should adopt our logs. However, these are in 1.5 and 1.6 :/.